packages/flutter_tools/lib/src/base/analyze_size.dart

```diff
@@ -38,6 +89,7 @@ class SizeAnalyzer {
   Future<Map<String, dynamic>> analyzeApkSizeAndAotSnapshot({
     @required File apk,
     @required File aotSnapshot,
+    @required File precompilerTrace,
   }) async {
     logger.printStatus('▒' * tableWidth);
     _printEntitySize(
```

Comment:
Can you help us fix the (new) line 107 for windows as well?

Reply:
not sure what you mean?

Reply:
You already did in ab1c196